uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/tabs/job-contract.js

```diff
@@ -1,90 +1,71 @@
 var Promise = require('es6-promise').Promise;
 var tab = require('./tab');
 
-module.exports = (function () {
-  return tab.extend({
-    tabTitle: 'Job Contract',
+module.exports = tab.extend({
+  tabTitle: 'Job Contract',
 
-    /**
-     * Clicks on the delete button
-     *
-     * @return {object}
-     */
-    attemptDelete: function () {
-      var casper = this.casper;
+  /**
+   * Clicks on the delete button
+   *
+   * @return {object}
+   */
+  attemptDelete: function () {
+    this.chromy.click('.hrjc-list-contract-item:nth-child(1) .btn-danger');
+    this.waitForModal();
+  },
 
-      casper.then(function () {
-        casper.click('.hrjc-list-contract-item:nth-child(1) .btn-danger');
-        this.waitForModal();
-      }.bind(this));
-    },
+  /**
+   * Opens the modal of an already existing contract
+   *
+   * @param  {string} mode "correct" or "revision"
+   * @return {Promise} resolves with the job contract modal object
+   */
+  openContractModal: function (mode) {
+    var param = mode === 'correct' ? 'edit' : (mode === 'revision' ? 'change' : '');
 
-    /**
-     * Opens the modal of an already existing contract
-     *
-     * @param  {string} mode "correct" or "revision"
-     * @return {Promise} resolves with the job contract modal object
-     */
-    openContractModal: function (mode) {
-      var casper = this.casper;
-      var param;
+    return new Promise(function (resolve) {
+      this.chromy.click('[ng-click="modalContract(\'' + param + '\')"]');
+      resolve(this.waitForModal('job-contract'));
```

Comment:
Please add an empty line above. Please resolve it in all other places in this PR.